Common/File/FileUtil.cpp

```diff
@@ -645,6 +739,11 @@ bool CreateEmptyFile(const Path &filename) {
 
 // Deletes the given directory and anything under it. Returns true on success.
 bool DeleteDirRecursively(const Path &directory) {	
+	if (Android_IsContentUri(directory.ToString())) {
```

Comment:
Should this check type?  Maybe REPORT?

-[Unknown]

Reply:
Looks like I've forgotten to update this one to the new ways, I'll fix it up.

Reply:
I'll just rewrite the whole function to be independent of path type.